scala/lambdas/entity-event-generator-lambda/src/main/scala/uk/gov/nationalarchives/entityeventgenerator/Lambda.scala

```diff
@@ -65,45 +65,50 @@ class Lambda extends LambdaRunner[ScheduledEvent, Int, Config, Dependencies] {
   ): IO[Int] = {
     val ignoredEventTypes = IgnoredEventTypes.values.map(_.toString)
     for {
+      _ <- logger.info(s"Getting entities for triggered time $eventTriggeredDatetime")
       updatedSinceResponses <- dADynamoDBClient.getItems[GetItemsResponse, PartitionKey](
         List(PartitionKey("LastPolled")),
         config.lastEventActionTableName
       )
       updatedSinceResponse = updatedSinceResponses.head
```

Comment:
Since we only want 1 result, should we add a limit on this `getItems` query that will only get one item?

Reply:
Doesn't seem worth it really. We know there's only one row in there.